cli/batch/service.py

```diff
@@ -45,3 +45,23 @@ def get(self, id):
             return None, str(error)
 
         return batch.get('data'), None
+
+    def get_status(self, id):
+        """
+        Get a batch status by its identifier.
+
+        Arguments:
+            id (string, required): batch identifier.
+        """
+        try:
+            result = loop.run_until_complete(
```

Comment:
In one line, if possible.
```suggestion
            batch_status = loop.run_until_complete(
```

Reply:
Done.